code/lib/telemetry/src/get-framework-info.ts

```diff
@@ -2,21 +2,23 @@ import type { PackageJson, StorybookConfig } from '@storybook/types';
 import { getActualPackageJson } from './package-json';
 
 const knownRenderers = [
-  'html',
-  'react',
-  'svelte',
-  'vue3',
-  'preact',
-  'server',
-  'vue',
-  'web-components',
-  'angular',
-  'ember',
+  '@storybook/html',
+  '@storybook/react',
+  '@storybook/svelte',
+  '@storybook/vue3',
+  '@storybook/preact',
+  '@storybook/server',
+  '@storybook/vue',
+  '@storybook/web-components',
+  '@storybook/angular',
+  '@storybook/ember',
+  'storybook-solidjs',
+  '@builder.io/qwik', // The Qwik framework is a renderer too, so the best we can do here is to detect the actual Qwik package
```

Comment:
no, but the framework package (`storybook-framework-qwik`) doesn't depend on any Storybook renderer, because itself is a renderer. It does however depend on `@builder.io/qwik`. And the logic in this file looks for what the frameworks depend on, to find the renderer.

Reply:
It looks to me like it looks for the renderer package, and in cases where the framework is also a renderer (e.g. `@storybook/angular`), it looks for that. Or am I missing something?

Reply:
Aha I see what you mean, I think you're right. I had missed the `[name]: version,` line which indeed takes the current framework into account as well.

```suggestion
  'storybook-framework-qwik', // The Qwik framework is a renderer too
```